cypress/e2e/CMS/$Biography.feature

```diff
@@ -0,0 +1,174 @@
+            | Last Name                   | LastName                                     | field_last_name                |
+            | Prefix                      | Dr.                                          | field_prefix                   |
+            | Suffix                      | M.S., Ph.D.                                  | field_suffix                   |
+            | Title (value 1)             | Title 1                                      | field_title[0][value]          |
+            | Organization 1 Name         | NCI                                          | field_org_name_1               |
+            | Organization 1 URL          | https://www.cancer.gov/                      | field_org_url_1                |
+            | Organization 2 Name         | DCEG                                         | field_org_name_2               |
+            | Organization 2 URL          | https://dceg.cancer.gov/                     | field_org_url_2                |
+            | Building/Office Room Number | Building 31, Room 7492                       | field_office_location          |
+            | Phone Number                | 555-555-5555                                 | field_phone_number             |
+            | Email Address               | test@test.com                                | field_email_address            |
+            | LinkedIn Profile Handle     | profilename                                  | field_linkedin_handle          |
+            | Twitter Profile Handle      | profilename                                  | field_twitter_handle           |
+        And user enters "https://www.ncbi.nlm.nih.gov/" into "Scientific Publications" text field                
+        And user clicks on "Add another item" to a title section
+        And user fills out the following fields
+            | fieldLabel      | value   | field_name            |
+            | Title (value 2) | Title 2 | field_title[1][value] |
+        And user fills out "Body" text area with "This is a description of biography content type."
+        And user selects "NCI Shady Grove Campus" from "Campus" dropdown
+        And user selects 1 Biography Image from the list of images
+        And user remembers the source of selected biography image for further verification
+        And user selects "Posted Date" checkbox
+        And user selects "Reviewed Date" checkbox
+        And user selects "Updated Date" checkbox
+        And user selects "Published" from "Save as" dropdown
+        When user saves the content page
+
+    Scenario: Verify newly created content
+        Given user is navigating to the front end site with path site section plus "biography"
+        Then page title is "Full Name"
+        And "Updated" date is displaying today's date
+        And description reads "This is a description of biography content type."
+       # And the Biography image is matching the earlier selected image
```

Comment:
why this is commented out? It means you did not verify is this step actually works